### DISCUSSION TITLE
apply filter to mutiple cards

### DISCUSSION THREAD

**Discussion by sophiepabstci** [h2oai/wave#942] [Q&A]
Hi there! Just wanted to ask how to apply a filter to mutiple cards at the same time? What if the cards use different queries/data tables? Can you set relationships? Has anyone tried this yet? Thanks  a lot in advance for your support, Sophie

**Comment by mturoci** [comment]
Hi @sophiepabstci thanks for the question. Yes, this is definitely possible. Let's say you have 2 tables, both with different data. What you need to do is to create 3rd card with your desired filters (textbox, dropdown etc.), listen for their value changes and update the data for the 2 tables respectively.
